### PR TITLE
Features & Discussion & stuff

### PR DESCRIPTION
**Pull request by AntimatterReactor**
just some suggestion to update(?) the logo

### CODE REVIEW

**Comment by DynamicSquid** [comment]
why're they all 69ights lol?

**Comment by AntimatterReactor** [comment]
> why're they all 69ights lol?

wha... it was an N just like the logo3.png except shadowed... But If you don't think so, i will probably resize it or create a new one, I guess

**Comment by AntimatterReactor** [comment]
I mean, I dunno, I have updated a crap ton of stuff but, I don't understand the code except for parts like opening file.

**Comment by DynamicSquid** [comment]
oh I actually like it! I haven't really though about support for when the user wants to build it from source since I'm planning to have to user code on a website, but that's actually a neat idea. since this is a big change, I'm going to put this on hold for now if that's okay since I still have a few more things I want to get done, but definitely in a later update I'm going to add support for a source build. thanks!

**Comment by DynamicSquid** [comment]
I also noticed that the binary will contain the .exe file, but I think it would be better if the user compiles the program themselves...

File: README.md [hunk 1]
```diff
@@ -50,19 +50,61 @@ int number = add(2, 3) + 4;
 print(number + "\n");
 ```
 
+## How to Use
+
+1. Open cmd, bash, or powershell
+2. Type:
+
+```bash
+%PathToNightFolder%/src/Night.exe %filepath%.night
```
Comment (DynamicSquid) [inline README.md:59]: I think it would be better if the user compiles this themselves

File: src/Night.cpp [hunk 2]
```diff
@@ -4,10 +4,15 @@
...
+int main(int argc, char *argv[])
 {
 	try {
-		ExtractCode();
+		if (argc > 0){
+			ExtractCode(argv[1]);
+		} else {
+			char *sdefault = (char *)"source.night";
+			ExtractCode(sdefault);
+		}
```
Comment (DynamicSquid) [inline src/Night.cpp:15]:
```cpp
if (argc > 0)
    ExtractCode(argv[1]);
else
    ExtractCode((char*)"source.night");
```

**Comment by AntimatterReactor** [comment]
Right... Alright... I'll fix those. Thanks for the compliment, too.

**Comment by AntimatterReactor** [comment]
And... Boom! Conflict Resolved.

**Comment by DynamicSquid** [comment]
oh yeah sorry about the conflicts they were my fault. also about when I want to merge this, I still want to get 2 or 3 things done with my language first before I merge this, so it might take 1 to 2 weeks if that's alright

**Comment by AntimatterReactor** [comment]
That's obviously alright

**Comment by HahaYes-dev** [comment]
HahaYes here... i think I need a pfp

**Comment by johnstev111** [comment]
@HahaYes Well you change your pfp by changing it

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by johnstev111** [comment]
What happens when you use your email... duh my anti-virus hates me

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by AntimatterReactor** [comment]
uhh did i make a mistake? it seems that... there's some conflict?

**Comment by DynamicSquid** [comment]
Eh don't worry thats my bad. Just ignore it, ill fix it soon

**Comment by DynamicSquid** [comment]
quick question, is it okay if I put your tutorial on the website instead of github?

**Comment by AntimatterReactor** [comment]
That'd be fine, anyway, pretty random, but I made a naive compiler for Night language ~~, please ignore, it's bad, im serious~~

**Comment by AntimatterReactor** [comment]
Anyway, the conflicts are now getting worse...

**Comment by AntimatterReactor** [comment]
Should I just stop the pull request or clone your master branch, a.k.a removing my progress?

**Comment by DynamicSquid** [comment]
oh yeah don't worry about it , it's minor stuff. I actually already merged part of your code for the source build. I just need to extract the tutorial and everything will be done!

**Comment by AntimatterReactor** [comment]
ok then, good to hear that.

**Comment by AntimatterReactor** [comment]
Gotta do something

**Pull request closed without merging**